src/mumble/Settings.cpp

```diff
@@ -674,6 +674,8 @@ void Settings::load(QSettings* settings_ptr) {
 	// Config updates
 	SAVELOAD(uiUpdateCounter, "lastupdate");
 
+	SAVELOAD(qsDatabasePath, "database");
```

Comment:
```suggestion
	SAVELOAD(qsDatabasePath, "databaseLocation");
```
Maybe this way it'd be clearer what exactly this field is for